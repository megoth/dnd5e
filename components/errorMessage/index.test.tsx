import React from "react";
import { render } from "@testing-library/react";
import ErrorMessage from "./index";

describe("ErrorMessage", () => {
  it("renders", () => {
    const { asFragment } = render(
      <ErrorMessage error={new Error("test 1")}>test 2</ErrorMessage>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
