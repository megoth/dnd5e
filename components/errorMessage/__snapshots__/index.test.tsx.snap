// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage render a fallback when app is not available 1`] = `
<DocumentFragment>
  <div
    data-testid="error"
  >
    <h1
      data-testid="error-title"
    >
      error2
    </h1>
  </div>
</DocumentFragment>
`;

exports[`ErrorMessage renders 1`] = `
<DocumentFragment>
  <div
    data-testid="error"
  >
    <h1
      data-testid="error-title"
    >
      Translated error message
    </h1>
  </div>
</DocumentFragment>
`;
